app/templates/components/provider-carousel.hbs

```diff
@@ -5,19 +5,27 @@
             <div class={{if (not index) "item active" "item"}}>
                 {{#each slide as |provider index|}}
                     <div class={{if (not index) (concat "col-sm-2 " columnOffset) "col-sm-2"}}>
-                        {{#if lightLogo}}
-                            <a href="/preprints/{{provider.id}}"
+                        {{#if (and provider.domain provider.domainRedirectEnabled)}}
```

Comment:
I'm not sure this logic should be in the template. Should probably create a computed property for this.